scripts/test.ps1

```diff
@@ -225,17 +227,17 @@ function Invoke-Test
                 $testFilter = "/testCaseFilter:`"$TPT_TestFilter`""
             }
 
-            if($fx -eq $TPT_TargetFrameworkCore20)
+            if($fx -eq $TPT_TargetFrameworkNet60)
             {
                 $vstestConsoleFileName = "vstest.console.dll"
                 $targetRunTime = ""
-                $vstestConsolePath = Join-Path (Get-PackageDirectory $TPT_TargetFrameworkCore20 $targetRuntime) $vstestConsoleFileName
+                $vstestConsolePath = Join-Path (Get-PackageDirectory $TPT_TargetFrameworkCore21 $targetRuntime) $vstestConsoleFileName
```

Comment:
Before (L104) you have replaced core20 with net6, here it's netcoreapp2.1, are both set as expected?

Reply:
Yes, vstest.console builds for net451, the change to net48 is only for test projects.